src/enums/authorized-actions-e.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * List of actions authorized via roles. See Authorizations module.
+ * Ultimately, this will be combined into the Authorization Roles enum.
+ */
+export enum AuthorizedActionsE {
+  CONVERSION_FILING,
+  CORRECTION_FILING,
+  ENABLE_DOCUMENT_RECORDS,
+  FREEZE_UNFREEZE_FILING,
+  NO_CONTACT_INFO,
+  RESTORATION_FILING,
+  SPECIAL_RESOLUTION_FILING,
+  STAFF_BREADCRUMBS,
+  STAFF_COMMENTS,
+  STAFF_FILING
+}
```

Comment:
Also need to add:
- AGM_EXTENSION_FILING
- AGM_CHG_LOCATION_FILING
- ALTERATION_FILING
- AMALGAMATION_FILING
- CONSENT_AMALGAMATION_OUT_FILING
- CONSENT_CONTINUATION_OUT_FILING
- DELAY_DISSOLUTION_FILING (will be used in near future)
- FIRM_CONVERSION_FILING

Also, please add conditionals in code for the above.

Ref: https://docs.google.com/spreadsheets/d/13P4ZA19lJfXJX95N5OfQMu6jYuotDH6h2hDf2DnAv0E/edit?gid=1335805235#gid=1335805235

Reply:
Looks good.